gitlab/datadog_checks/gitlab/metrics.py

```diff
@@ -51,8 +51,8 @@
     "gitlab_transaction_rails_queue_duration_total": "transaction.rails_queue_duration_total",
     "gitlab_transaction_view_duration_total": "transaction.view_duration_total",
     "gitlab_view_rendering_duration_seconds": "view_rendering_duration_seconds",
-    "http_requests_total": "http_requests_total",
-    "http_request_duration_seconds": "http_request_duration_seconds",
+    "http_requests_total": "http.requests_total",
+    "http_request_duration_seconds": "http.request_duration_seconds",
```

Comment:
Can we compare the metric value we get for the new metric and legacy equivalent? Are they the same? If so, could we  remove the duplicate from the `allowed_metrics` list?

Reply:
Sounds good! I'll prefix these with `rack` since they are tracking rack requests.